.pipelines/singletenancy/cilium/cilium-e2e-step-template.yaml

```diff
@@ -94,26 +133,13 @@ steps:
     name: "validatePods"
     displayName: "Validate Pods"
 
-  - script: |
```

Comment:
Could you explain why we remove this (here and in other places)? Don't we still want to test cns restarting, instead of just node restarts?

Reply:
CNS restart is a part of the `make test-load` testing suite. No need to do it a second time imo.